packages/odyssey-react-mui/src/BackgroundContext.tsx

```diff
@@ -0,0 +1,142 @@
+/*!
```

Comment:
In keeping with best practices, this file should be named `<something>Provider`.

Reply:
updated it to `ContrastModeProvider`.